website/docs/d/backup_recoveries.html.markdown

```diff
@@ -0,0 +1,571 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_backup_recoveries"
+description: |-
+  Get information about List of Recoveries.
+subcategory: "IBM Backup Recovery API"
```

Comment:
can we remove the word API from subcategory this names is use to group the res/ds in registry docs.

can we add this allowed-subcategoreis since its new service